lib/decompress/zstd_decompress.c

```diff
@@ -901,7 +901,7 @@ static size_t ZSTD_decompressFrame(ZSTD_DCtx* dctx,
         switch(blockProperties.blockType)
         {
         case bt_compressed:
-            decodedSize = ZSTD_decompressBlock_internal(dctx, op, (size_t)(oend-op), ip, cBlockSize, /* frame */ 1);
+            decodedSize = ZSTD_decompressBlock_internal(dctx, op, (size_t)(oend-op), ip, cBlockSize, /* frame */ 1, 0);
```

Comment:
API : 
the signification of this last parameter is not immediately obvious.
Prefer using `enum` types, 
it will give a proper name at calling sites, helping code self-documentation.